yarn-project/end-to-end/src/e2e_crowdfunding_and_claim.test.ts

```diff
@@ -56,7 +56,6 @@ describe('e2e_crowdfunding_and_claim', () => {
   let deadline: number; // end of crowdfunding period
 
   let uintNote!: any;
```

Comment:
Changes in this test are mainly related to [this comment](https://github.com/AztecProtocol/aztec-packages/pull/11904#discussion_r1954122823).